string id
string state
string dump_errors
string error
string procedure_result
float32 input_pos
float32 pos_abs
float32 pos_rel
float32 input_vel
float32 vel_estimate
